README.md

```diff
@@ -124,6 +124,19 @@ steps:
       - docker#v3.0.1
 ```
 
+Additional `docker build` arguments be passed via the `additional-build-args` setting:
+
+```yaml
+steps:
+  - command: 'echo amaze'
+    env:
+      ARG_1: wow
+    plugins:
+      - seek-oss/docker-ecr-cache#v1.3.0:
+          additional-build-args: '--ssh= default=\$SSH_AUTH_SOCK'
```

Comment:
Guessing it's not worth structuring it up like

```yaml
additional-build-args:
  - ssh: default=\$SSH_AUTH_SOCK
```

Reply:
nah couldn't be bothered with that, plus might break things if you want `-short flags` etc